### PR TITLE
Add Presenters Section

### LINKED ISSUE #314: Add "presented by" section
We want to add a section to the website that lists the presenters of the event. That's simplabs, Joschka Kintscher, and Balint Erdi.

We should add that section below the sponsors section with

* title: "Presenters"
* content: text "EmberFest is presented by", images of the 3 presenters (our logo, photos for Joschka and Balint), each one with a web link and Twitter profile link

Overall, that section can be visually the same as the "Speakers" section.

### PR DESCRIPTION
**Pull request by inesilva**
It closes #314

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,6 +79,53 @@ <h4>Melanie Sumner</h4>
     </div>
   </section>
 
+  <section class="presenters">
```
Comment (oscard0m) [inline index.html:82]: > We should add that section below the sponsors' section with

originally posted in the issue: https://github.com/emberfest/emberfest.github.io/issues/314

File: index.html [hunk 2]
```diff
@@ -79,6 +79,53 @@ <h4>Melanie Sumner</h4>
...
+              ><img src="/images/icons/twitter-blue.svg"
+            /></a>
+            <a href="https://balinterdi.com/"
+              ><img src="/images/icons/link-blue.svg"
+            /></a>
+          </div>
+        </li>
+
+        <li>
+          <img src="/images/sponsors/simplabs.svg" class="photo logo" />
```
Comment (oscard0m) [inline index.html:114]: 🎨 We are already saying `simplabs` in L115. What do you think of using just simplabs logo, without the text?

image

File: index.html [hunk 3]
```diff
@@ -79,6 +79,53 @@ <h4>Melanie Sumner</h4>
...
+          <h4>Simplabs</h4>
```
Comment (oscard0m) [inline index.html:115]:
```suggestion
          <h4>simplabs</h4>
```

**Review by oscard0m — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (inesilva) [reply index.html:114]: I think it is a good idea

File: index.html (hunk 1 above)
Comment (inesilva) [reply index.html:82]: oh I didn't notice, thanks!

**Review by jkintscher — DISMISSED** [review]
Please replace Joachim, who’s left EmberFest in 2015, with my pic/links 🙏😅

**Comment by inesilva** [comment]
> Please replace Joachim, who’s left EmberFest in 2015, with my pic/links 🙏😅

sorry! thanks for letting me know 😅

File: index.html (hunk 1 above)
Comment (oscard0m) [reply index.html:114]: Any idea on how to differentiate the presenters' section from the sponsors' section? A change of background maybe?

(I personally preferred to put the section below the speakers as you suggested initially... but maybe there's a reason to not do so)

**Review by oscard0m — APPROVED** [review]
Changes look good.

I personally don't like how the presenters look after the sponsors. I prefer what you suggested before (after the speakers) but... is a requirement of the task. Let's see what other reviewers say.

**Comment by balinterdi** [comment]
> I personally don't like how the presenters look after the sponsors. I prefer what you suggested before (after the speakers) but... is a requirement of the task. Let's see what other reviewers say.

I agree, I think there should be more visual differentiation. Why is it a requirement of the task (or, where is this written)?

> Any idea on how to differentiate the presenters' section from the sponsors' section? A change of background maybe?

I'm not a designer, but a separate section with a different background color could work, I think, yes.

Also, we do we want to say "presenters"? We've so far used the word "organizers", though not officially. At any rate, that's not a hill I'm doing to die on, it's just that organizers sounds more natural to me, that's all.

**Comment by oscard0m** [comment]
> > I personally don't like how the presenters look after the sponsors. I prefer what you suggested before (after the speakers) but... is a requirement of the task. Let's see what other reviewers say.
> 
> I agree, I think there should be more visual differentiation. Why is it a requirement of the task (or, where is this written)?

https://github.com/emberfest/emberfest.github.io/issues/314#issue-1348178215

> > Any idea on how to differentiate the presenters' section from the sponsors' section? A change of background maybe?
> 
> I'm not a designer, but a separate section with a different background color could work, I think, yes.
> 
> Also, we do we want to say "presenters"? We've so far used the word "organizers", though not officially. At any rate, that's not a hill I'm doing to die on, it's just that organizers sounds more natural to me, that's all.

I like 'organizers' as well.

**Comment by balinterdi** [comment]
Right, but Marco's comment doesn't say that it needs to be blended into the Sponsors section just above it:

> Overall, that section can be visually the same as the "Speakers" section.

I think he refers more to the layout which you guys have done just fine. Discerning it visually from the section above it is fine (and, I'd add, better).

**Comment by oscard0m** [comment]
> Right, but Marco's comment doesn't say that it needs to be blended into the Sponsors section just above it:

@balinterdi I got confused by this sentence but I agree is not very clear.
> **Note**
> We should add that section below the sponsors section - _Marco_

---

@inesilva I would add it under 'speakers' section as @balinterdi suggests.

**Comment by inesilva** [comment]
I agree with putting it back under the speakers.
Regarding the naming, I don't have a strong opinion; it is more about what word represents you best @balinterdi

**Review by balinterdi — APPROVED** [review]
Thank you, Inês, looks good!

**Pull request merged**